incubator-tools/extending_entity_bounding_boxes/extending_entity_bounding_box.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>This needs to provide a more precise description of what the notebook does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
added more description on tool in the updated notebook.